Source/Workers/decodeDraco.js

```diff
@@ -82,6 +84,21 @@ define([
                         componentDatatype : ComponentDatatype.fromTypedArray(vertexArray)
                     }
                 };
+
+                transform = new draco.AttributeQuantizationTransform();
```

Comment:
It should be ok to remove the `var transform;` above and have this be `var transform = new draco.AttributeQuantizationTransform();`